confirm.h

```diff
@@ -27,8 +29,19 @@ struct confirm_query {/*{{{*/
 		};
 	};
 
-	uint16_t icmpid; /* icmpid == 0 fixes reverse flow id =revflow */
-			 /* uint16_t icmpseq used to identify probes */
+	union {
+		struct {
+			uint16_t src_port;
+			uint16_t dst_port;
+			uint32_t ack_number;
+			uint8_t  control_flags;
+			uint16_t window;
+			uint16_t urgent_pointer;
+		} tcp;
+
+		uint16_t icmpid; /* icmpid == 0 fixes reverse flow id =revflow */
```

Comment:
Yes, I think it is not the same indeed. For TCP we also accept flowid but it is just the probe number that we use to match the response.